src/ORM/DataObjectSchema.php

```diff
@@ -302,6 +302,10 @@ protected function buildTableName($class)
 
         // Generate default table name
         if (!$table) {
+            trigger_error(
```

Comment:
We don't need this warning for non-namespaced classes (e.g. `Page`) so can you add a check for this first?

Reply:
Good point, will add that in